lua/autorun/wire_load.lua

```diff
@@ -25,6 +25,8 @@ if SERVER then
 	-- this file
 	AddCSLuaFile("autorun/wire_load.lua")
 
+	util.AddNetworkString( "IsEditing" )
```

Comment:
I recommend moving this to the same file as the E2 functions. Makes more sense to have it be right next to that code